tests/operation/test_io.py

```diff
@@ -0,0 +1,69 @@
+import io
+import builtins
+import contextlib
+from unittest import mock
+
+from dffml.df.base import op
+from dffml.operation.output import GetSingle
+from dffml.df.memory import MemoryOrchestrator
+from dffml.util.asynctestcase import AsyncTestCase
+from dffml.df.types import DataFlow, Input, Definition
+from dffml.operation.io import AcceptUserInput, printOutput
+
+
+class TestInputOutput(AsyncTestCase):
+    async def setUp(self):
+        self.stdout = io.StringIO()
```

Comment:
I can't remember, do we need to call `super().setUp()` here? If there's a `setUp()` in AsyncTestCase then we should. If not then don't worry about it.

Reply:
Other than that this file looks great..

Reply:
Yes there is `setUp()` in AsyncTestCase.